core/src/main/java/com/novoda/noplayer/exoplayer/forwarder/BitrateForwarder.java

```diff
@@ -0,0 +1,57 @@
+package com.novoda.noplayer.exoplayer.forwarder;
+
+import com.google.android.exoplayer2.C;
+import com.google.android.exoplayer2.Format;
+import com.google.android.exoplayer2.source.AdaptiveMediaSourceEventListener;
+import com.google.android.exoplayer2.upstream.DataSpec;
+import com.novoda.noplayer.listeners.BitrateChangedListeners;
+
+import java.io.IOException;
+
+class BitrateForwarder implements AdaptiveMediaSourceEventListener {
```

Comment:
No longer extending from the abstract `ExoPlayerEventForwarder`, instead, we map directly from the `ExoPlayer` events to the `no-player` events.